blueprints/app/files/bower.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
This is an ergonomics question, paired with https://github.com/ember-cli/ember-cli/pull/6698#discussion_r96703772 ... a blank `bower.json` file means that users can blissfully use `bower`. Otherwise the user is presented with the above message asking them to `bower init` which is _way_ more than they need.